geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/KeywordCommand.scala

```diff
@@ -0,0 +1,92 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo.commands
+
+import com.beust.jcommander._
+import com.beust.jcommander.converters.IParameterSplitter
+import org.geotools.data.DataStoreFinder
+import org.locationtech.geomesa.tools.accumulo.commands.KeywordCommand.KeywordParameters
+import org.locationtech.geomesa.tools.accumulo.{DataStoreHelper, GeoMesaConnectionParams}
+import org.locationtech.geomesa.tools.common.FeatureTypeNameParam
+import org.locationtech.geomesa.tools.common.commands.Command
+
+import scala.collection.JavaConversions._
+import scala.io.StdIn
+
+class KeywordCommand(parent: JCommander) extends Command(parent) {
+  import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType._
+  override val command: String = "keyword"
+  override val params = new KeywordParameters()
+
+  override def execute(): Unit = {
+    val dsParams = new DataStoreHelper(params).paramMap
+    val tryDs = DataStoreFinder.getDataStore(dsParams)
+    if (tryDs == null) {
+      throw new ParameterException("Could not load a data store with the provided parameters")
+    }
+
+    val sft = tryDs.getSchema(params.featureName)
+
+    if (params.keywordsToAdd != null) {
+      sft.addKeywords(params.keywordsToAdd.mkString(KEYWORDS_DELIMITER))
+    }
+
+    if (params.keywordsToRemove != null) {
+      sft.removeKeywords(params.keywordsToRemove.mkString(KEYWORDS_DELIMITER))
```

Comment:
That actually is an issue that I accounted for with removeAllKeywords but forgot about the case when all are manually deleted. 
When the retrieved keywords is an empty string, (which i'm guessing is somehow encoded with a null byte) an additional empty keyword is retrieved.
We could handle this by deleting the key value pair when removing the last keyword. Or maybe switching to another un-used delimiter.